src/it/scala/com/exasol/cloudetl/kinesis/KinesisImportQueryGeneratorIT.scala

```diff
@@ -87,14 +131,85 @@ class KinesisImportQueryGeneratorIT
     assertResultSet(expected2)
   }
 
-  private def assertResultSet(expected: List[(Int, Int, String, String, Boolean)]): Unit = {
+  test("KinesisImportQueryGenerator imports all data types") {
+    val streamName = "Stream_3"
+    createKinesisStream(streamName, 1)
+    putRecordSWithAllDataTypesIntoStream(streamName)
+    createTableWithAllDataTypes()
+    executeKinesisPathScriptWithConnection(streamName)
+    val expected = List(
+      (
+        "[\"first\",\"second\"]",
+        true,
+        10,
+        10.55,
+        "{\"firstNestedValue\":10,\"secondNestedValue\":\"second\"}",
+        null,
+        "shardId-000000000000",
+        true
+      )
+    )
+    assertResultSetAllDataTypes(expected)
+  }
+
+  private[kinesis] def putRecordSWithAllDataTypesIntoStream(
+    streamName: String
+  ): Unit = {
+    val partitionKey = "shardId-000000000000"
+    val recordData =
+      s"""{\"array_val\": [\"first\", \"second\"],
+         | \"bool_val\": true,
+         | \"int_val\": 10,
+         | \"double_val\": 10.55,
+         | \"object_val\": {\"firstNestedValue\": 10,\"secondNestedValue\": \"second\"},
+         | \"string_val\": null
+         | }""".stripMargin.replace("\n", "")
```

Comment:
As above, remove escapes.

Reply:
And all other places where string interpolation is used.

Reply:
done